src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Outcome.cshtml

```diff
@@ -116,34 +417,71 @@
+                                    Successful - already active
                                 </label>
                             </div>
                             <div class="govuk-radios__item">
                                 <input class="govuk-radios__input"
-                                       type="radio"
-                                       id="OptionNo"
-                                       name="Status"
-                                       value=@OversightReviewStatus.Unsuccessful
-                                       data-aria-controls="conditional-OptionNo"
-                                       @radioCheckedUnsuccessful />
+                                    type="radio"
+                                    id="OptionSuccessfulFitnessForFunding"
+                                    name="OversightStatus"
+                                    value=@OversightReviewStatus.SuccessfulFitnessForFunding
+                                    data-aria-controls="conditional-OptionSuccessfulFitnessForFunding"
+                                    @radioCheckedSuccessfulFitnessForFunding />
+                                <label class="govuk-label govuk-radios__label" for="OptionSuccessfulFitnessForFunding">
+                                    Successful - fitness for funding
+                                </label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input"
+                                    type="radio"
+                                    id="OptionNo"
+                                    name="OversightStatus"
+                                    value=@OversightReviewStatus.Unsuccessful
+                                    data-aria-controls="conditional-OptionNo"
+                                    @radioCheckedUnsuccessful />
                                 <label class="govuk-label govuk-radios__label" for="OptionNo">
-                                    Unsuccessful application
+                                    Unsuccessful
+                                </label>
+                            </div>
+                            <div class="govuk-radios__divider">or</div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input"
+                                    type="radio"
+                                    id="OptionInProgress"
+                                    name="OversightStatus"
+                                    value=@OversightReviewStatus.InProgress
+                                    data-aria-controls="conditional-OptionInProgress"
+                                    @radioCheckedInProgress />
+                                <label class="govuk-label govuk-radios__label" for="OptionInProgress">
+                                    Set to 'in progress'
```

Comment:
There is a issue with the design here. We don't store or display it anywhere as 'in progress' . It'll always be captialized